ibtk/include/ibtk/HierarchyMathOps.h

```diff
@@ -1155,6 +1186,26 @@ class HierarchyMathOps : public SAMRAI::tbox::DescribedClass
                     int src2_idx = invalid_index,
                     SAMRAI::tbox::Pointer<SAMRAI::pdat::SideVariable<NDIM, double> > src2_var = nullptr);
 
+    /*!
+     * \brief Compute dst = alpha grad coef1 (div src1)  + beta src2, the variable coefficient
+     * dilatational stress force.
+     *
+     * \see setPatchHierarchy
+     * \see resetLevels
+     */
+    void vc_dilatational(int dst_idx,
+                         SAMRAI::tbox::Pointer<SAMRAI::pdat::SideVariable<NDIM, double> > dst_var,
+                         double alpha,
+                         int coef1_idx,
```

Comment:
Here the order of the arguments is good, but I am wondering if we can merge together `alpha` and `coef1`.